src/Tablet/Dashboard/DashboardTab.js

```diff
@@ -106,18 +107,12 @@ const DashboardTab = ({ dashboard }) => {
   }
 
   return (
-    <div
-      style={{
-        display: "inline-grid",
-        gridTemplateColumns: "auto auto auto auto auto auto auto auto",
-        gridTemplateRows: "auto auto auto auto",
-        gridGap: 0,
-        gridAutoRows: "minmax(256px, auto)",
-        gridAutoFlow: "dense",
-      }}
-    >
-      {tiles.current}
-    </div>
+    <IonGrid>
+      <IonRow justify-content-center align-items-center>
+        <IonCol size="12">{tiles.current}</IonCol>
```

Comment:
I'm not sure this is right. This really means that all the tiles will be in the single full-width column of the single row of a grid.